src/shared/client/cg_api.cpp

```diff
@@ -54,15 +54,15 @@ int trap_CM_MarkFragments( int numPoints, const vec3_t *points, const vec3_t pro
 
 	std::vector<std::array<float, 3>> mypoints(numPoints);
 	std::array<float, 3> myproj;
-	memcpy((float*)mypoints.data(), points, sizeof(float) * 3 * numPoints);
+	memcpy(mypoints.data(), points, sizeof(float) * 3 * numPoints);
```

Comment:
This one may be a good candidate for a custom `memcpy` with a check if there is no alternative than `memcpy`.

Reply:
Right. In this case the types are incompatible according to C++ rules, but we know the layout is the same.